### PR TITLE
Fix PHPUnit tests across all packages

### PR DESCRIPTION
**Pull request by JoelAlphonso**
Fix PHPUnit tests across all packages with the option to run tests from the monorepo package (either all packages at once or individually) or run tests from each package if their dependencies are installed.

### Bug Fixes

* **phpcs:** exit execution after -l option ([54fc393](https://github.com/charcoalphp/charcoal/commit/54fc3939abf3dc69664baa64e2327978e9ef1bd3))

### CODE REVIEW

File: monorepo-builder.php [hunk 1]
```diff
@@ -24,6 +24,25 @@
...
+            'locomotivemtl/charcoal-email' => '*',
+            'locomotivemtl/charcoal-factory' => '*',
+            'locomotivemtl/charcoal-image' => '*',
+            'locomotivemtl/charcoal-object' => '*',
+            'locomotivemtl/charcoal-property' => '*',
+            'locomotivemtl/charcoal-queue' => '*',
+            'locomotivemtl/charcoal-translator' => '*',
+            'locomotivemtl/charcoal-ui' => '*',
+            'locomotivemtl/charcoal-user' => '*',
+            'locomotivemtl/charcoal-view' => '*'
```
Comment (mcaskill) [inline monorepo-builder.php:44]:
```suggestion
            'locomotivemtl/charcoal-view' => '*',
```

**Review by mcaskill — CHANGES_REQUESTED** [review]
Looks good as far as I can see from skimming most files.
There's some checks that will need to be resolved, I think, though I have not consulted them all. Some suggestions for the PHPCS script will improve CI formatting for humans to read.

Repository-wide suggestions:

* The opening brace for the methods must go on its own line.
    ```diff
    - protected function setUp(): void    {
    + protected function setUp(): void
    + {
    ```
* Rename `tests/script` directory to `tests/scripts`.

File: packages/translator/composer.json [hunk 2]
```diff
@@ -31,7 +31,8 @@
         "phpunit/phpunit": "^9.5",
         "squizlabs/php_codesniffer": "^3.5",
         "php-coveralls/php-coveralls": "^2.2",
-        "charcoal/admin": "^3.2"
+        "charcoal/admin": "^3.2",
+        "charcoal/app": "*"
```
[the remark this answers is not in the document]
Comment (JoelAlphonso) [reply packages/translator/composer.json:35]: why not

File: packages/app/src/Charcoal/App/Module/ModuleInterface.php [hunk 3]
```diff
@@ -10,5 +10,5 @@ interface ModuleInterface
     /**
      * @return void
      */
-    public function setup();
+    protected function setUp(): void
```
[the remark this answers is not in the document]
Comment (JoelAlphonso) [reply packages/app/src/Charcoal/App/Module/ModuleInterface.php]: It was fixed by the time you posted it

File: packages/cms/src/Charcoal/Cms/TemplateableTrait.php [hunk 4]
```diff
@@ -31,7 +31,7 @@ trait TemplateableTrait
      *
      * @var string
      */
-    protected $templateIdent;
+    protected $templateIdent = '';
```
[the remark this answers is not in the document]
Comment (JoelAlphonso) [reply packages/cms/src/Charcoal/Cms/TemplateableTrait.php:34]: Will push that in the next commit

File: packages/view/src/Charcoal/View/ViewableInterface.php [hunk 5]
```diff
@@ -31,7 +31,7 @@ public function setTemplateIdent(string $templateIdent);
      *
      * @return string
      */
-    public function templateIdent(): string;
+    public function templateIdent(): ?string;
```
[the remark this answers is not in the document]
Comment (JoelAlphonso) [reply packages/view/src/Charcoal/View/ViewableInterface.php:34]: Will push that in the next commit

File: packages/image/src/Charcoal/Image/Imagick/Effect/ImagickBlurEffect.php [hunk 6]
```diff
@@ -47,7 +47,7 @@ public function processRadial()
     {
         $angle = $this->angle();
         $channel = $this->image()->imagickChannel($this->channel());
-        $this->image()->imagick()->radialBlurImage(($angle), $channel);
+        $this->image()->imagick()->rotationalBlurImage(($angle), $channel);
```
[the remark this answers is not in the document]
Comment (JoelAlphonso) [reply packages/image/src/Charcoal/Image/Imagick/Effect/ImagickBlurEffect.php:50]: Issue created

File: packages/image/src/Charcoal/Image/Imagemagick/Effect/ImagemagickBlurEffect.php [hunk 7]
```diff
@@ -48,7 +48,7 @@ public function processMotion()
     public function processRadial()
     {
         $channel = $this->image()->convertChannel($this->channel());
-        $cmd = '-channel ' . $channel . ' -radial-blur ' . $this->angle();
+        $cmd = '-channel ' . $channel . ' -rotational-blur ' . $this->angle();
```
[the remark this answers is not in the document]
Comment (JoelAlphonso) [reply packages/image/src/Charcoal/Image/Imagemagick/Effect/ImagemagickBlurEffect.php:51]: Issue created

**Review by mcaskill — APPROVED** [review]

**Pull request merged**